src/Illuminate/Mail/Transport/Transport.php

```diff
@@ -67,4 +67,19 @@ protected function beforeSendPerformed(Swift_Mime_Message $message)
             }
         }
     }
+
+    /**
+     * Get the number of recipients.
+     *
+     * @param  \Swift_Mime_Message  $message
+     * @return int
+     */
+    protected function getCount(Swift_Mime_Message $message)
+    {
+        $contacts = array_merge(
+            (array) $message->getTo(), (array) $message->getCc(), (array) $message->getBcc()
+        );
+
+        return count($contacts);
```

Comment:
Yeh, I know. But there's little else we can do if the service in question just says "yes, i've queued that for you".